model_zoo/app_integrate_bundle/README.md

```diff
@@ -22,12 +22,12 @@ The script is tested with:
 You can conda environments to install the dependencies.
 
 ```bash
-pip install monai==0.9.1 scikit-learn==0.24.2
+pip install monai==1.2.0 scikit-learn==0.24.2
```

Comment:
I would suggest to remove the MONAI versions here, just:
`pip install monai scikit-learn==0.24.2`
It can avoid the pip errors before MONAI 1.2 release and avoid future upgrade.
What do you think?

Thanks.